app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+
+  def create
+    resource = User::Create.call(user: sign_up_params)
+    yield resource if block_given?
+    if resource.persisted?
+      handle_login(resource)
+    else
+      clean_up_passwords resource
+      set_minimum_password_length
+      respond_with resource
+    end
+  end
+
+  protected
```

Comment:
I suggest changing `protected` to `private`. It does not really matter in that case, but I would do it just to be consistent with the rest of our controllers.